oaknational/Oak-Web-Application
# feat: remove fuzziness from elastic search query [LESQ-554]

## Description

Music year: 2014

- Removes the fuzziness parameter from elastic search query on the 2023 api

## Issue(s)

Fixes #
Content being served that doesn't match well ahead of legacy content

## How to test

1. Go to https://deploy-preview-2175--oak-web-application.netlify.thenational.academy
2. Search for 'animal farm' for example
3. You should see a single 'new' result that matches the query, ahead of the actual animal farm lessons

## Screenshots

How it used to look (delete if n/a):
<img src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/ac65ec53-d424-4a57-af06-6d5e31c688f4" width=500>


How it should now look:
<img src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/bd6c58ed-0185-4328-a9f4-baf9f8dec2ef" width=500>
